src/main/java/org/apache/commons/exec/CommandLine.java

```diff
@@ -398,6 +398,7 @@ private static String[] translateCommandline(final String toProcess) {
      *
      * @param dirtyExecutable the executable
      * @return the platform-specific executable string
+     * @throws IllegalArgumentException If dirtyExecutable is null or empty

```

Comment:
Hello @ArdenL-Liu 
Thank you for your PR.
You might as well update the other Javadoc instances in this class where IllegalArgumentException is thrown but not documented, especially since we seem to document some but not all cases.

Reply:
Hi garydgregory,
I reviewed CommandLine.java  again, and added the javadoc to the constructor CommandLine(String).
you can review it.
thanks.